assets/src/scss/layout/_country-selector.scss

```diff
@@ -0,0 +1,192 @@
+
+#country-selector {
+  _-- {
+    color: inherit;
+  }
+  font-size: $font-size-sm;
+  margin-bottom: 24px;
+  text-align: start;
+  width: 100%;
+
+  a _-- {
+    color: inherit;
+  }
+
+  a:hover,
```

Comment:
We can probably remove these selectors for hover and focus, because they set the same value as the generic ones, and use the same variable.

Reply:
I removed the `a` one, but the generic styles we have for `.btn` are a lot more specific than just those border/outline fixes, that would lead to more specific declarations in the end